<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F1F1F1;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cad_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
border-color:#FFFFFF;
border-width:1px;
background: #968C8C;
}

</string>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
border-color:#FFFFFF;
border-width:1px;
background: #968C8C;
}
</string>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upt_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
border-color:#FFFFFF;
border-width:1px;
background: #968C8C;
}
</string>
   </property>
   <property name="text">
    <string>ATUALIZAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dell_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
border-color:#FFFFFF;
border-width:1px;
background: #968C8C;
}
</string>
   </property>
   <property name="text">
    <string>DELETAR</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>641</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QHeaderView::section{
background: #968C8C;
padding:2px;
color:white;
border: 1px solid #fffff8;
font-size:10pt;
}

QTableWidget{
gridline-color: black;
font-size:11pt;
}

QTableWidget QTableCornerButton::section{
	background-color: rgb(255, 0, 0);
border:1pxsolid #ff0000;
}</string>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOME</string>
    </property>
    <property name="background">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E-MAIL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SEXO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PERMISSION</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:5px;
border-color: Black;
padding:10px;
border-width:1px;
border : 2px solid black	;
}

</string>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:5px;
border-color: Black;
padding:10px;
border-width:1px;
border : 2px solid black	;
}
QLineEdit:hover{
border : 2px solid #1E90FF	;

}
</string>
   </property>
   <property name="placeholderText">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:5px;
border-color: Black;
padding:10px;
border-width:1px;
border : 2px solid black	;
}
QLineEdit:hover{
border : 2px solid #1E90FF	;

}</string>
   </property>
   <property name="placeholderText">
    <string>E-mail </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="senha">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:5px;
border-color: Black;
padding:10px;
border-width:1px;
border : 2px solid black	;
}
QLineEdit:hover{
border : 2px solid #1E90FF	;

}</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_upt_user">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
border-color:#FFFFFF;
border-width:1px;
background: #968C8C;
}</string>
   </property>
   <property name="text">
    <string>ATUALIZAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>STATUS: </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdb_femi">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Feminino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>GÃªnero:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdb_masc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Masculino </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:5px;
border-color: Black;
padding:10px;
border-width:1px;
border : 2px solid black	;
}
QComboBox:hover{
border : 2px solid #1E90FF	;

}</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>PermissÃ£o</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>PREENCHER DADOS</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>cad_btn</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>upt_btn</zorder>
  <zorder>dell_btn</zorder>
  <zorder>tableWidget</zorder>
  <zorder>id</zorder>
  <zorder>nome</zorder>
  <zorder>email</zorder>
  <zorder>senha</zorder>
  <zorder>label_2</zorder>
  <zorder>btn_upt_user</zorder>
  <zorder>label_3</zorder>
  <zorder>rdb_femi</zorder>
  <zorder>label_5</zorder>
  <zorder>rdb_masc</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
